basicsr/losses/loss_util.py

```diff
@@ -93,3 +94,31 @@ def wrapper(pred, target, weight=None, reduction='mean', **kwargs):
         return loss
 
     return wrapper
+
+
+def get_local_weights(residual, ksize):
+
```

Comment:
Since this function is only used by `get_refined_artifact_map`, it is better to add a docstring to specify it.